app.rb

```diff
@@ -27,7 +27,15 @@
 
 # get_times
 get '/times' do
-  erb :times
+  erb :times, locals: { times: ts.get_times }
+end
+
+get '/times/:time' do
+  data = []
+  ts.get_times.each do |t|
```

Comment:
You can fetch an individual time by uuid, just include the uuid: paramter, assuming rimesync behave like pymesync